clients/web/src/utilities/MiscFunctions.js

```diff
@@ -36,24 +36,22 @@ girder.formatDate = function (datestr, resolution) {
 };
 
 /**
- * Format a size in bytes into a human-readable string with metric unit prefixes.
+ * Format a size in bytes into a human-readable string with metric unit
+ * prefixes.
  */
 girder.formatSize = function (sizeBytes) {
-    // If it's > 1GB, report to two decimal places, otherwise just one.
-    var precision = sizeBytes > 1073741824 ? 2 : 1;
```

Comment:
The reason for this logic was that it made it a bit easier to see progress of uploads on files on the order of GB. In that case, if we report to only 1 decimal place, the number only changed every ~100MB, which can take a very long time while uploading.  Maybe this is OK though.